style.css

```diff
@@ -198,6 +198,282 @@ body {
+  padding-left: 57px;
+  padding-right: 61px;
+  padding-top: 43px;
+  width: 488px;
+}
+
+.order__title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
+  margin: 0;
+}
+
+.order__subtitle {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: 0.07em;
+  margin-bottom: 61px;
+  color: #000;
+  opacity: 0.5;
+  text-transform: uppercase;
+}
+
+.order__description {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  margin-bottom: 35px;
+  color: #000;
+  opacity: 0.5;
+}
+
+.color-list {
+  display: inline-flex;
+  flex-direction: row;
+  height: 38px;
+  list-style-type: none;
+  margin-bottom: 53px;
+  margin-right: 13px;
+  transform: translateX(-39px);
+}
+
+.list__item {
+  border-radius: 50%;
+  margin-right: 12px;
+}
+
+.list__item_grey,
+.list__item_blue,
+.list__item_green {
+  width: 30.18px;
+  height: 30px;
+  margin-top: 4px;
+}
+
+.list__item_grey {
+  background: #dedede;
+}
+
+.list__item_pink {
+  width: 38px;
+  height: 38px;
+  background: #f29f9f;
+}
+
+.list__item_blue {
+  background: #a8bff9;
+}
+
+.list__item_green {
+  background: #90bd8f;
+}
+
+.order__price {
+  display: inline-block;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 56px;
+  transform: translateY(-5px);
+}
+
+.order-number {
+  border: 1.5px solid #000;
+  display: inline-block;
+  width: 100px;
+  height: 44px;
+  margin-bottom: 30px;
+  margin-right: 10px;
+  padding-left: 8px;
+  padding-top: 10px;
+}
+
+.number__input,
+.minus,
+.plus {
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 0px;
+}
+
+.minus {
+  margin-left: 8px;
+  margin-right: 18px;
+}
+
+.number__input {
+  background: transparent;
+  border: none;
+  margin-right: 11px;
+  width: 12px;
+}
+
+.order__cart {
+  background: transparent;
+  border: 1.5px solid #000;
+  display: inline-block;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  margin-bottom: 45px;
+  height: 44px;
+  padding-right: 11px;
+  text-transform: uppercase;
+  width: 253px;
+}
+
+.range-1 {
+  display: inline-block;
+  position: relative;
+  transform: translateX(2px);
+}
+
+.range-1__link {
+  color: #000;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  margin-right: 10px;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.outfit {
+  background: linear-gradient(to right, #ebe3d8 27%, #000 27% 32%, #ebe3d8 32%);
+  display: flex;
+  padding-bottom: 140px;
+  padding-left: 88px;
+  padding-right: 148px;
+  padding-top: 139px;
+  width: 100%;
+}
+
+.outfit__img {
+  display: flex;
+  z-index: 1;
+}
+
+.outfit-details {
+  display: flex;
+  flex-direction: column;
+  margin-top: 116px;
+  z-index: 2;
+}
+
+.outfit-content {
+  background-color: #FAE8E4;
```

Comment:
Codacy found an issue: [Expected "#FAE8E4" to be "#fae8e4" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)